modules/nodes-nodes-resources-configuring-about.adoc

```diff
@@ -29,65 +28,35 @@ introduction of allocatable resources.
-dedicated cgroup hierarchy separate from system daemons.
-
-Optionally, the node can be made to enforce kube-reserved and system-reserved by
-specifying those tokens in the enforce-node-allocatable flag.  If specified, the
-corresponding `--kube-reserved-cgroup` or `--system-reserved-cgroup` needs to be provided.
-In future releases, the node and container runtime will be packaged in a common cgroup
-separate from `system.slice`.  Until that time, we do not recommend users
-change the default value of enforce-node-allocatable flag.
-
-Administrators should treat system daemons similar to Guaranteed pods.  System daemons
-can burst within their bounding control groups and this behavior needs to be managed
-as part of cluster deployments.  Enforcing system-reserved limits
-can lead to critical system services being CPU starved or OOM killed on the node. The
-recommendation is to enforce system-reserved only if operators have profiled their nodes
-exhaustively to determine precise estimates and are confident in their ability to
-recover if any process in that group is OOM killed.
-
-As a result, we strongly recommended that users only enforce node allocatable for
-`pods` by default, and set aside appropriate reservations for system daemons to maintain
-overall node reliability.
+The node is able to limit the total amount of resources that pods can consume based on the configured allocatable value. This feature significantly improves the reliability of the node by preventing pods from using CPU and memory resources that are needed by system services such as the container runtime and node agent. To improve node reliability, administrators should reserve resources based on a target for resource use.
+
+The node enforces resource constraints using a new cgroup hierarchy that enforces quality of service. All pods are launched in a dedicated cgroup hierarchy that is separate from system daemons.
+
+Administrators should treat system daemons similar to pods that have a guaranteed quality of service. System daemons can burst within their bounding control groups and this behavior must be managed as part of cluster deployments. Reserve CPU and memory resources for system daemons by specifying the amount of CPU and memory resources to reserve in `system-reserved`.
+
+Enforcing `system-reserved` limits can prevent critical system services from receiving CPU and memory resources. As a result, a critical system service can be ended by the out-of-memory killer. The recommendation is to enforce `system-reserved` only if you have profiled the nodes exhaustively to determine precise estimates and you are confident that critical system services can recover if any process in that group is ended by the out-of-memory killer.
+
+As a result, we strongly recommended that you only enforce node allocatable for `pods` by default, and set aside appropriate reservations for system daemons to maintain node reliability.
```

Comment:
I know this is existing text (which I added!), any thoughts on what this means? _only enforce node allocatable for `pods` by default_

_only enforce node allocatable for pods, which is the default_?? 

See: 
https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#enforcing-node-allocatable

Reply:
No mark-up for _pods_.

Reply:
The `pods` markup boo-boo is because I tried to reuse the words as-is from 3.11.  In that release, there was a short procedure to set kubelet command-line arguments and the value was `pods`.  https://docs.openshift.com/container-platform/3.11/admin_guide/allocating_node_resources.html#node-enforcement-1

I can't find a procedure in 4.6 that covers setting kubelet command-line arguments.  Please ping me here or in slack and let me know if you think that para should be removed.

Reply:
I'm not sure we have such a procedure, or if it's even needed. I personally don't like the references to setting flags. Seems odd to have two ways of setting parameters, flags vs config files. It probably makes sense to remove that paragraph as we don't explain how to set the flag/parameter.

Reply:
Thanks @mburke5678, I removed the "As a result..." para.